allennlp/tests/models/language_model_test.py

```diff
@@ -86,6 +87,8 @@ def setUp(self):
 
         self.expected_embedding_shape = (2, 8, 14)
         self.bidirectional = True
+        self.result_keys = {"loss", "forward_loss", "backward_loss", "lm_embeddings",
```

Comment:
`self.result_keys.add("backward_loss")` might call attention to the difference a bit better, but this is fine too.